stubs/networkx/networkx/algorithms/centrality/betweenness_subset.pyi

```diff
@@ -1,8 +1,11 @@
-from _typeshed import Incomplete
-
+from networkx.classes.graph import Graph, _Edge, _Node
 from networkx.utils.backends import _dispatch
 
 @_dispatch
-def betweenness_centrality_subset(G, sources, targets, normalized: bool = False, weight: Incomplete | None = None): ...
+def betweenness_centrality_subset(
+    G: Graph[_Node], sources: list[_Node], targets: list[_Node], normalized: bool = False, weight: str | None = None
```

Comment:
One thing we do in typeshed is using more lenient annotations in arg positions, if possible:

```suggestion
    G: Graph[_Node], sources: Iterable[_Node], targets: Iterable[_Node], normalized: bool = False, weight: str | None = None
```

`Iterable` needs to be imported from `collections.abc`.